master/reference/calicoctl/resources/policy.md

```diff
@@ -69,14 +70,17 @@ spec:
  | No                    | Yes                  | `egress`            |
  | Yes                   | Yes                  | `ingress, egress`   |
 
-\*\* The `doNotTrack` and `preDNAT` fields are meaningful only when applying policy to a
+\*\* The `doNotTrack` and `preDNAT` and `applyOnForward` fields are meaningful only when applying policy to a
 [host endpoint]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/hostendpoint).
-Only one of them may be set to `true` (in a given policy). If they are both `false`, or when applying the policy to a
+
+Only one of `doNotTrack` and `preDNAT` may be set to `true` (in a given policy). If they are both `false`, or when applying the policy to a
 [workload endpoint]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/workloadendpoint),
 the policy is enforced after connection tracking and any DNAT.
 
+`applyOnForward` must be set to `true` if either `doNotTrack` or `preDNAT` is `true` because for a given policy, any untracked rules or rules before DNAT will in practise apply to forwarded traffic.
```

Comment:
We plan to standardize on the American spellings, sorry! So then "practise" --> "practice"